ui_framework/components/_mixins.scss

```diff
@@ -174,15 +174,8 @@
   }
 }
 
-/**
- * 1. Angular will add an ng-untouched class to an input if it hasn't been touched yet.
- *    We only want invalid inputs to appear invalid after the user has had a chance to interact
- *    with it.
- */
 @mixin formControlInvalid {
```

Comment:
Could you add a comment here to make sure we don't re-introduce Angular classes in the future?

```scss
/**
 * We specifically don't include Angular's ng-${state} classes here because we don't want to be tightly
 * coupled with Angular.
 */
```